src/Drush/Commands/ShorthandCommands.php

```
@@ -0,0 +1,117 @@
+        'key' => 'filename',
+      ]);
+      if (empty($usedShorthands[$story])) {
+        // Remove the story folder.
+        $folder = $destination_uri . '/' . $story;
+        $this->fileSystem->deleteRecursive($folder);
+        $this->writeln($this->t('Removed shorthand: @title', [
+          '@title' => $story,
+        ]));
+      }
+      else {
+        foreach ($storyVariants as $storyVariant => $data) {
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$data'. (undefined)

(UnusedLocalVariable)